src/gf-oemid

```diff
@@ -66,7 +71,11 @@ coreos_gf_run_mount "${tmp_dest}"
 # * grub config
 # * BLS config (for subsequent config regeneration)
 # First, the grub config.
-grubcfg_path=/boot/loader/grub.cfg
+if [ "$(arch)" == "x86_64" ]; then
+    grubcfg_path=/boot/loader/grub.cfg
+else
+    grubcfg_path=/boot/efi/EFI/fedora/grub.cfg
+fi
```

Comment:
I think rather than looking at the arch we could here instead do e.g.:

`if coreos_gf exists "/boot/efi/EFI"; then`

Reply:
Ok